auth/service.bal

```diff
@@ -0,0 +1,101 @@
+import ballerina/http;
+import ballerina/crypto;
+import ballerina/uuid;
+import e_backend.data;
+import ballerina/persist;
+import ballerina/sql;
+
+listener http:Listener voterListener = new (9090);
+
+final data:Client dbClient = check new ();
+
+
+public type VoterLogin record { 
+    string nationalId; 
+    string password; 
+};
+
+service /Voter\-registration/api/v1 on voterListener {
```

Comment:
Currently, what you have done is, having different ballerina packages for `auth`, `election-config` etc. I think we can use different modules within one ballerina package to represent them.